html/semantics/embedded-content/the-iframe-element/srcdoc_change_hash.html

```diff
@@ -6,6 +6,8 @@
 <script src="/resources/testdriver.js"></script>
 <script src="/resources/testdriver-vendor.js"></script>
 <script>
+  setup({ single_test: true });
```

Comment:
I'd prefer https://github.com/web-platform-tests/wpt/pull/19926 to this change.

Reply:
Okay. I've merged your pull request and removed all modifications from this changeset so it will merge cleanly.